faker/providers/address/__init__.py

```diff
@@ -75,8 +76,13 @@ def address(self):
     def country(self):
         return self.random_element(self.countries)
 
-    def country_code(self):
-        return self.random_element(self.country_codes)
+    def country_code(self, representation='alpha-2'):
```

Comment:
can you declare `'alpha-2'` and `'alpha-3'` constants on the provider, and reference the constant instead of using a string?

```python
class Provider(BasePRovider):
    ALPHA_2 = 'alpha-2'
    ALPHA_3 = 'alpha-3'

    def country_code(self, representation=ALPHA_2):
        if representation == self.ALPHA_2:
            # ...
```